easybuild/easyconfigs/f/FSL/FSL-5.0.11-foss-2018b.eb

```diff
@@ -0,0 +1,40 @@
+name = 'FSL'
+version = '5.0.11'
+
+homepage = 'http://www.fmrib.ox.ac.uk/fsl/'
+description = """FSL is a comprehensive library of analysis tools for FMRI, MRI and DTI brain imaging data."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ["http://www.fmrib.ox.ac.uk/fsldownloads/"]
+sources = ['%(namelower)s-%(version)s-sources.tar.gz']
+patches = [
+    'FSL-%(version)s_makefile_fixes.patch',
+    'FSL-5.0.9_missing_lib.patch',
+    'FSL-%(version)s_build_extras.patch',
+]
+checksums = [
+    '8a76d36ff280ab3bfa79c46838a1baf7ea8ffe21d0b84bc7546cb8a1cef46c57',  # fsl-5.0.11-sources.tar.gz
+    'dcf76c2f54eaa729c300a917794a5cd3c37dc46205a7b415eb866e1a871f2dbf',  # FSL-5.0.11_makefile_fixes.patch
+    '10cd2d6c66ee0488244036743e70d211dbbdfadac651e5c95dd1de1f676a4b38',  # FSL-5.0.9_missing_lib.patch
+    '9f29713c4c9610fc9beb20b08f2c3a90499a5d17ec57f6beb6193530fe121b46',  # FSL-5.0.11_build_extras.patch
+]
+
+# Depends on
+# - https://github.com/easybuilders/easybuild-easyconfigs/pull/6700
```

Comment:
@crubb No need to include this in the easyconfig file, just mentioning it in the description is fine